src/rtk/features/singleRights/singleRightsSlice.ts

```diff
@@ -206,7 +206,7 @@ const singleRightSlice = createSlice({
             serviceWithStatus.status = 'PartiallyDelegable';
           } else {
             serviceWithStatus.status = 'NotDelegable';
-            serviceWithStatus.errorCode = action.payload[0].details[0].code;
+            serviceWithStatus.errorCodes?.push(action.payload[0].details[0].code);
```

Comment:
Why push when this is only done once?
Why not just set it equal to `action.payload[0].details[0].code`?

Is it because `action.payload[0].details[0].code` is not a list? If so, then I'm wondering why  `serviceWithStatus.errorCodes` is a list, if it only contains one object

Reply:
Changed the solution so this is not here anymore